### PR TITLE
XRT-2639: Added OPC-UA Alarm configuration

### PR DESCRIPTION
**Pull request by BinduRao2018**

### CODE REVIEW

File: DeviceServices/opc-ua/commands/start_device_sim.sh [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+VER=1.3
 OPC_UA_SIM=iotechsys/opc-ua-sim:$VER
 OPC_UA_LDS=iotechsys/opc-ua-lds:$VER
 
 docker pull $OPC_UA_LDS
 docker pull $OPC_UA_SIM
 
 docker run --rm -d --network=host --name opc-ua-lds $OPC_UA_LDS
-docker run --rm -d --network=host --name opc-ua-sim $OPC_UA_SIM -l /example-scripts/simulation.lua --discovery-url=opc.tcp://$(hostname):4840
+docker run --rm -d --network=host --name opc-ua-sim $OPC_UA_SIM --discovery-url=opc.tcp://$(hostname):4840
```
Comment (dpazj) [inline DeviceServices/opc-ua/commands/start_device_sim.sh]: I think this should be reverted as the other examples still require the simulation script to be run?

File: DeviceServices/opc-ua/alarms/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## Trigger the alarm
+
+
+```bash
+../commands/put_request_alarm.sh
+```
+The event is notified on the configured _telemetry topic_.
+
+_To know more on the event management and event format, see [OPC-UA Event Registration Component](https://docs.iotechsys.com/edge-xrt22/extension-components/opc-ua-event-registration-component.html)._
```
Comment (dpazj) [inline DeviceServices/opc-ua/alarms/README.md]:
```suggestion
_For more information about event management, see [OPC-UA Event Registration Component](https://docs.iotechsys.com/edge-xrt22/extension-components/opc-ua-event-registration-component.html)._ 
```

File: DeviceServices/opc-ua/alarms/README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+> **Note** Xrt must be run from this context as the configuration files use relative pathnames
+
+### Alarm
+
+Alarm is a type of an event in OPC UA with a state condition. 
+
+## Configure Alarm for a resource
+
+Below command will setup an alarm to a monitored resource.
```
Comment (dpazj) [inline DeviceServices/opc-ua/alarms/README.md]:
```suggestion
The below command will register the monitoring of events on a resource. 
```

File: DeviceServices/opc-ua/alarms/README.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+```bash
+../commands/event_register.sh
+```
+
+## Trigger the alarm
+
```
Comment (dpazj) [inline DeviceServices/opc-ua/alarms/README.md]: Add some more info about why this triggers the alarm - 
"In this example, the condition is configured to produce an event when it's severity changes. This command will change the severity of the condition and will trigger an event"

File: DeviceServices/opc-ua/README.md [hunk 5]
```diff
@@ -2,10 +2,13 @@
 
 ## Overview
 
-This page shows you how to setup and run the OPC-UA device service example.
+This page shows you how to setup and run the OPC-UA device service examples.
 
 For more information about the Device Service please review the [OPC-UA Device Service](https://docs.iotechsys.com/edge-xrt22/device-service-components/opc-ua-device-service-component.html) documentation.
 
+A separate example configuration [alarms](./alarms) has been added to demonstrate OPC-UA alarms.
```
Comment (dpazj) [inline DeviceServices/opc-ua/README.md]: Perhaps we can reword this to: 
A separate example configuration [alarms](./alarms) has been added to demonstrate the OPC-UA device service events registration component.

**Review by dpazj — CHANGES_REQUESTED** [review]
We refer to this example as "alarms", however I think it might be best to make it clear that this an example for the `events_registration component` - which allows you to monitor events/conditions/alarms on an opc-ua server.

File: DeviceServices/opc-ua/commands/start_device_sim.sh (hunk 1 above)
Comment (BinduRao2018) [reply DeviceServices/opc-ua/commands/start_device_sim.sh]: Updated

File: DeviceServices/opc-ua/alarms/README.md (hunk 2 above)
Comment (BinduRao2018) [reply DeviceServices/opc-ua/alarms/README.md]: Addressed.

File: DeviceServices/opc-ua/alarms/README.md (hunk 3 above)
Comment (BinduRao2018) [reply DeviceServices/opc-ua/alarms/README.md]: Addressed.

File: DeviceServices/opc-ua/alarms/README.md (hunk 4 above)
Comment (BinduRao2018) [reply DeviceServices/opc-ua/alarms/README.md]: Updated.

File: DeviceServices/opc-ua/README.md (hunk 5 above)
Comment (BinduRao2018) [reply DeviceServices/opc-ua/README.md]: Updated. The new example config is under event-registration instead of Alarms as suggested above.

**Review by dpazj — APPROVED** [review]
LGTM
